libraries/AC_WPNav/AC_Loiter.cpp

```diff
@@ -96,7 +96,7 @@ void AC_Loiter::init_target(const Vector3f& position)
 
     // initialise desired acceleration and angles to zero to remain on station
     _predicted_accel.zero();
-    _desired_accel = _predicted_accel;
```

Comment:
these two changes in AC_Loiter look like they're not triggered from standby mode so they will always apply.  Are these unrelated changes?  .. and the remove of the set_max_speed_xy() and set_max_accel_xy() is intentional?

Reply:
Yes I will remove them.

Reply:
Removed them